app/Resources/views/pagelayout.html.twig

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en-US"
+      class="no-js">
```

Comment:
I disagree. This is an accepted standard for code formatting, and is one of the many religious wars amongst the tabs-vs-spaces crowd. I agree that we need to have *a* standard, and I strongly recommend that we follow this one. It is the format I recommend to any team working with HTML in a VCS as well, as it makes for nice clean diffs when a single attribute is changed. It can be automated using tidy: http://api.html-tidy.org/tidy/quickref_5.2.0.html#indent-attributes

We can discuss further, but for now just be aware that it was done very much on purpose.